### PR TITLE
Add onboarding photos page

### PR DESCRIPTION
**Pull request by t-tanveer**
Installed this library for image picker https://github.com/react-native-image-picker/react-native-image-picker

Instructions said to use yarn, but I had to do both yarn and npm installs for it to work properly. But perhaps yarn is unnecessary?

Currently images are optional, user may proceed to next page without uploading an image. Can change as per requirements.

Tested on Emulator only (can't find usb hub atm, but will try to test on physical device)
image

### CODE REVIEW

**Comment by jeverd** [comment]
Does this work on IOS? @t-tanveer

**Comment by t-tanveer** [comment]
> Does this work on IOS? @t-tanveer

I don't have an iPhone or iOS development setup so I'm not sure. LMK on discord if there's a way I can still test.
The library and the options of the library that I'm using are supposedly android and iOS supported. But yes, it should tested on actual devices. 
@jeverd

**Comment by jeverd** [comment]
@t-tanveer np, will need to have ios setup locally to test. I think there's probably some permissions things we need to add to get this to work on IOS.  Could you take a look at the library and see if that's the case? If it is, can you add it in and then we can get one of the others who have IOS setup to test.

**Comment by t-tanveer** [comment]
Yes sure, there is information on [the library repo](url) on permissions for iOS and android

Namely

```Add the appropriate keys to your Info.plist,
If you are allowing user to select image/video from photos, add NSPhotoLibraryUsageDescription.```

Will try and push a commit to add that later. If someone else is trying to test it in meanwhile, check above link

**Review by lnogueir — COMMENTED** [review]
LGTM !! Good job, bro.
Please, also just add a check to make sure at least 1 image is selected.
And before we merge, let's just make sure we have all the permissions necessary for iOS.

File: src/screens/OnboardingPhotosScreen.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  View,
+  TouchableOpacity,
+  Image,
+} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import ApproachText from '../ApproachText';
+import ApproachWideButton from '../ApproachWideButton';
+import {OnboardingLabelsScreenName} from './OnboardingLabelsScreen';
+import {launchImageLibrary} from 'react-native-image-picker';
+import addIcon from '../../assets/icons/add.jpg';
```
Comment (lnogueir) [inline src/screens/OnboardingPhotosScreen.js:14]: Capitalize images. Also, maybe `PlusIcon`? (rename file too)

File: src/screens/OnboardingPhotosScreen.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      includeBase64: true,
+    };
+
+    launchImageLibrary(options, response => {
+      if (response.error) {
+        setImagePickerError(true);
+      } else if (response.assets) {
+        setFunction({
+          uri: response.assets[0].uri,
+        });
```
Comment (jeverd) [inline src/screens/OnboardingPhotosScreen.js:46]: @t-tanveer can we add the logic to upload this image to firebase? Once you upload this to firebase, you should get a URL which you can pass around to: 
https://github.com/approach-me/app-v2/blob/6cfbad9bf49b3c77dd62f8ca03f6933f474cdfe2/src/screens/OnboardingFinishScreen.js#L51


This post is really good, I think if you follow this, you can set it up easily: https://stackoverflow.com/questions/59525278/react-native-firebase-storage-get-url-after-uploading-an-image

Comment (lnogueir) [reply src/screens/OnboardingPhotosScreen.js:46]: And FYI, in order to have the photos available in the `OnboardingFinishScreen` above, all you need to do is call setUserDetail({photos: <list of photo URLs here>})

**Comment by lnogueir** [comment]
Added the missing iOS file descriptors. Should be good to merge now.

**Review by jeverd — APPROVED** [review]

**Pull request merged**